examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -103,12 +105,6 @@ CHIP_ERROR CHIPCommand::MaybeSetUpStack()
     ReturnLogErrorOnFailure(mOperationalKeystore.Init(&mDefaultStorage));
     ReturnLogErrorOnFailure(mOpCertStore.Init(&mDefaultStorage));
 
-    // chip-tool uses a non-persistent keystore.
```

Comment:
@thivya-amazon  why are you removing this init?

Reply:
That was a mistake during conflict resolution.